config/config.yaml

```diff
@@ -84,6 +84,20 @@ environments:
     admin_creds: MasterOsAdminCredentials
     api_port: 8443
     # version: 3.6.1.44.16
+  ocp4:
+    # hosts: use OPENSHIFT_ENV_OSE_HOSTS=host:role1:...,host2:role1:...
+    hosts_type: SSHAccessibleHost
+    # this is the user for remote access to the OpenShift nodes
+    user: core
+    type: StaticEnvironment
+    user_manager: auto
+    # set users in OPENSHIFT_ENV_OSE_USER_MANAGER_USERS=user:password,...
+    # user_manager_users: joe:redhat,alice:redhat
+    #cli: MasterOsPerUserCliExecutor
+    cli: SharedLocalCliExecutor
+    admin_creds: MasterOsAdminCredentials
+    api_port: 6443
+    version: 4.0.0.0 # version not supported on cluster (yet)
```

Comment:
Hardcode `version: 4.0.0.0` LG currently for a quite period before 4.1 comes :)

Reply:
@xingxingxia , not having access to version anonymously will require some changes. I'd like to make sure we are not getting what we need before implementing something that may not be as nice as present implementation.